src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotInv = {};
+
+  for (const key in robot) {
+    // перевіряємо на дублювання властивостей в об'єкті
+    if (robotInv.hasOwnProperty(robot[key])) {
+      return null;
+    }
+    // додаємо в об'єкт властивість як ключ і ключ як властивість
```

Comment:
not needed to add comments :)

Nice code - code that don't need to be commented